component_catalog/models.py

```diff
@@ -2528,9 +2549,9 @@ def get_purldb_entries(self, user, max_request_call=0, timeout=10):
         Return the PurlDB entries that correspond to this Package instance.
 
         Matching on the following fields order:
```

Comment:
Let's add a small comment about the matching order being from the most to the least accurate field.